### PR TITLE
chore: add analytics info wording

### LINKED ISSUE #251: Edit prompt information
Edit prompt information in other PR.

_Originally posted by @tyyzqmf in https://github.com/awslabs/clickstream-analytics-on-aws/pull/241#discussion_r1335435375_

### PR DESCRIPTION
**Pull request by luorobin-a2z**
----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

## Summary

(describe what this merge request does)

closes #251 

## Implementation highlights

(describe how the merge request does for feature changes, share the RFC link if it has)

## Test checklist

- [ ] add new test cases
- [ ] all code changes are covered by unit tests
- [ ] end-to-end tests
  - [ ] deploy web console with CloudFront + S3 + API gateway
  - [ ] deploy web console within VPC
  - [ ] deploy ingestion server
    - [ ] with MSK sink
    - [ ] with KDS sink
    - [ ] with S3 sink
  - [ ] deploy data processing
  - [ ] deploy data modeling
    - [ ] new Redshift Serverless
    - [ ] provisioned Redshift
    - [ ] Athena
  - [ ] deploy with reporting

## Is it a breaking change

- [ ] add parameters without default value in stack
- [ ] introduce new service permission in stack
- [ ] introduce new top level stack module

## Miscellaneous

- [ ] introduce new symbol link source file(s) to be shared among infra code, web console frontend, and web console backend

### CODE REVIEW

File: frontend/public/locales/en-US/analytics.json [hunk 1]
```diff
@@ -26,20 +26,21 @@
...
+    "description": "You can create custom analysis and dashboard in a drag-and-drop manner in this module.",
     "comeSoon": "The feature is about to be launched."
   },
   "explore": {
     "title": "Explore",
-    "description": "Explore allows you query and analyze the clickstream data with an easy-to-use UI in a dynamic manner. Select an analytics model to begin with.",
+    "description": "The Explore module provides a range of advanced analytical models to enable you to dynamically query and analyze clickstream data. Select an analysis model to begin.",
     "analyticsModel": "Analytics model",
-    "exploitativeAnalytics": "Exploratory analytics",
+    "exploitativeAnalytics": "Exploratory analytsis",
```
Comment (zxkane) [inline frontend/public/locales/en-US/analytics.json:36]: -> analysis?

File: frontend/public/locales/en-US/help.json [hunk 2]
```diff
@@ -1,9 +1,98 @@
 {
-  "dashboard": {
+  "dashboardsInfo": {
     "title": "Dashboard",
-    "description": "Dashboard description",
+    "description": "A dashboard is a read-only snapshot of an analysis that you can share with other users for reporting purposes. By default, the solution provides an User Life Cycle dashboard to help you understand each stage in the customer jorney. You can also create custom dashboard by saving explorative analysis that you want to view regularly.",
```
Comment (zxkane) [inline frontend/public/locales/en-US/help.json:4]: an User -> a User
jorney -> journey

File: frontend/public/locales/en-US/help.json [hunk 3]
```diff
@@ -1,9 +1,98 @@
...
     "links": {
-      "dashboardDocLinkName": "What is dashboard?"
+      "docLinkName": "What is dashboard?",
+      "docLink": "https://awslabs.github.io/clickstream-analytics-on-aws/en/latest/dashboard/"
```
Comment (zxkane) [inline frontend/public/locales/en-US/help.json:7]: I prefer to build the documentation link like v1, using docs.aws.amazon.com for the English version. So specifying the relative here, then the frontend to generate the final link.

File: frontend/public/locales/en-US/help.json [hunk 4]
```diff
@@ -1,9 +1,98 @@
...
+    }
+  },
+  "userLifecycleInfo": {
+    "title": "User life cycle dashboard",
+    "description": "The solution comes with pre-built analytics dashboards to help you understand how users use your app or website, including new user acquisition, user engagement, events, retention, devices, and crash analysis。",
```
Comment (zxkane) [inline frontend/public/locales/en-US/help.json:12]: 。

File: frontend/public/locales/en-US/help.json [hunk 5]
```diff
@@ -1,9 +1,98 @@
...
+    "title": "Data management",
+    "description": "The data management module makes it easy for you to view and manage all events, event attributes, and user attributes by automatically scanning all the clickstream data in your app to generate metadata. You can also modify the display names, descriptions, and data dictionaries for events and attributes to facilitate your filtering use in the Explore Analysis module.",
+    "links": {
+      "docLinkName": "Analyzes",
+      "docLink": "https://awslabs.github.io/clickstream-analytics-on-aws/en/latest/analyticstudio/analyzes/"
+    }
+  },
+  "metadataEventInfo": {
+    "title": "Event",
+    "description": "'Event' refers to the clickstream data generated by a  user action within an app or website (such as clicking, visiting a page, etc.). Events has two categories: preset events, namely events automatically collected by the SDK, usually start with an underscore '_'; custom events, namely events define and report by app owner, and the report timing and business purpose of the event varies.",
```
Comment (zxkane) [inline frontend/public/locales/en-US/help.json:76]: Events has -> have
defined and reported

**Review by zxkane — CHANGES_REQUESTED** [review]

File: frontend/public/locales/en-US/help.json (hunk 3 above)
Comment (chenhaiyun) [reply frontend/public/locales/en-US/help.json:7]: `v1.0.x` or `v1.1.x` ? below is the document link sample:
https://awslabs.github.io/clickstream-analytics-on-aws/zh/1.0.x/solution-overview/
https://awslabs.github.io/clickstream-analytics-on-aws/zh/1.1.x/solution-overview/

Comment (chenhaiyun) [reply frontend/public/locales/en-US/help.json]: Or use `"solution_version": "v1",` in aws-exports.json?

Comment (zxkane) [reply frontend/public/locales/en-US/help.json]: Good proposal! Let's use the major and minor versions from `aws-exports.json`.

File: frontend/src/ts/url.ts [hunk 6]
```diff
@@ -62,13 +62,20 @@ export const GETTING_STARTED_LINK_EN = '/getting-started.html';
 export const SUBMMIT_ISSUE_LINK =
   'https://github.com/awslabs/clickstream-analytics-on-aws/issues';
 
+export const getSolutionVersion = () => {
+  const configJSONObj: ConfigType = JSON.parse(
+    localStorage.getItem(PROJECT_CONFIG_JSON) || '{}'
+  );
+  return configJSONObj?.solution_version ?? 'latest';
```
Comment (zxkane) [inline frontend/src/ts/url.ts:69]: The document is using `1.0.x` to match all 1.0 releases, like `1.0.0.`, `1.0.1`, `1.0.2`.

I think we can't directly use the version as a prefix.

**Review by zxkane — CHANGES_REQUESTED** [review]

File: frontend/src/ts/url.ts (hunk 6 above)
Comment (chenhaiyun) [reply frontend/src/ts/url.ts]: Fixed. And `1.1.x` to match all 1.1 releases, like `1.1.0`, `v1.1.1`, `1.1.2`.

**Review by zxkane — APPROVED** [review]

**Pull request merged**